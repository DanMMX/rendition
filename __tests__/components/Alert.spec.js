/* globals expect, test */
import React from 'react'
import renderer from 'react-test-renderer'
import Provider from '../../src/components/Provider'
import Alert from '../../src/components/Alert'

test('Alert renders correctly', () => {
  const component = renderer.create(
    <Provider>
      <Alert>An alert</Alert>
    </Provider>
  )
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
