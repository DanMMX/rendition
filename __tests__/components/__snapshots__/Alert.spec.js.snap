// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alert renders correctly 1`] = `
<div
  className="sc-bdVaJa jKopLr"
>
  <div
    className="sc-htoDjs gDCMNh sc-EHOje fbMNid"
  >
    <div>
      
      An alert
    </div>
  </div>
</div>
`;
